src/scenes/game.scene.ts

```diff
@@ -26,7 +27,53 @@ export class GameScene extends Scene {
   create() {
     this.image = this.matter.add.sprite(0, 0, 'foobar'); // Image name specified in boot.scene.ts
     this.cursorKeys = this.input.keyboard.createCursorKeys();
-    this.matter.world.setBounds(0, 0, getGameWidth(this), getGameHeight(this), 0, true, false, true);
+    this.matter.world.setBounds();
+    this.matter.add.mouseSpring();
+    
+    const bodies = new Matter(this).bodies;
+
+    const group = this.matter.world.nextGroup(true);
+
+    const bridge = this.matter.add.stack(160, 290, 15, 1, 0, 0, (x: number, y: number) => {
+      return bodies.rectangle(x - 20, y, 53, 20, {
+        collisionFilter: { group: group, category: 0, mask: 0 },
+        chamfer: { radius: 0 },
+        density: 0.005,
+        frictionAir: 0.05
+      });
+    });
+
+    this.matter.add.chain(bridge, 0.3, 0, -0.3, 0, {
+      stiffness: 1,
+      length: 0,
+      render: {
+        visible: false
+      }
+    });
+
+    this.matter.add.stack(250, 50, 6, 3, 0, 0, (x: number, y: number) => {
+      return bodies.rectangle(x, y, 50, 50, between(20, 40) as any);
+    });
+
+    this.matter.add.rectangle(30, 490, 220, 380, {
+      isStatic: true,
+      chamfer: { radius: 20 }
+    }),
+
+    this.matter.add.rectangle(770, 490, 220, 380, {
+      isStatic: true,
+      chamfer: { radius: 20 }
+    }),
+
+    this.matter.add.worldConstraint(bridge.bodies[0], 2, 0.9, {
+      pointA: { x: 140, y: 300 },
+      pointB: { x: -25, y: 0 }
+    });
+
+    this.matter.add.worldConstraint(bridge.bodies[bridge.bodies.length - 1], 2, 0.9, {
+      pointA: { x: 660, y: 300 },
+      pointB: { x: 25, y: 0 }
+    });
```

Comment:
All of the above is from this example:

https://phaser.io/examples/v3/view/physics/matterjs/bridge

There are some differences in doing this in TypeScript, such as how I access Matter.bodies.